pkg/connectors/protos/data_credential_response.proto

```diff
@@ -6,7 +6,27 @@ syntax = "proto3";
 package connectors;
 option java_package = "com.datmesh";
 
+message Credentials {
+    string access_key = 1;
+    string secret_key = 2;
+    string username = 3;
+    string password = 4;
+    string api_key = 5;
+    string resource_instance_id = 6;
+    string url = 7;
+    string bucket = 8;
+    string database = 9;
+    string host = 10;
+    string port = 11;
+    string ssl = 12;
+    string connection_id = 13;
+    string connection_path = 14;
+    string custom_credentials_json = 15; // this field will be used in case we are using Egeria as the catalog since egeria does not support the above fields at the moment. Also this field will be used in case of WKC also when remoteStatus is REMOTE_CUSTOM (when a local asset is uploaded in WKC GUI), then we get a credentials json from WKC. we are not sure which fields will be present. so this field will be sending the json as it is received from WKC.
+    bool is_remote = 16;
+}
```

Comment:
Most of this information describes DataSource. We have another proto for passing this information. 
I would include only the first five fields.

Reply:
Some thoughts are provided [here](https://github.com/IBM/the-mesh-for-data/pull/137#discussion_r524485252). Please let me know your views. Thanks.